RELEASE_NOTES.md

```diff
@@ -8,6 +8,21 @@ redirect_from:
 ---
 # Version history
 
+## Version 0.8.1
+
+April 11th 2025: 🧑‍🤝‍🧑 The 18th draft changes to community governance model.
+
+* The project has switched to a community governance model.
+  * Governance model new covers steering team, composition, responsibilities, meetings, decision making process, lazy consensus, voting, and code of Conduct.
+  * The website is relocated to a new domain [https://standard.publiccode.net/](https://www.standardforpubliccode.org/)
+  * SPDX copyright text updated for all project files.
+* Removed optional requirement in Document the code about tempting examples.
+* Critera texts has been slightly simplified, to be more easily understood, with less technical terms.
```

Comment:
```suggestion
* Criteria texts has been slightly simplified, to be more easily understood, with less technical terms.
```
Another small typo

Reply:
also, "Criteria texts _has_ been" should be "Criteria texts _have_ been"